src/workbench/SideBar.tsx

```diff
@@ -123,8 +124,42 @@ const SideBarContents = ({ panes, ...props }: SideBarContentsProps) => {
   }, [tab, setParams]);
   const width = "5rem";
   const brand = useDeployment();
+  const intl = useIntl();
   return (
     <Flex height="100%" direction="column" {...props} backgroundColor="gray.50">
+      <Flex
+        backgroundColor="brand.500"
+        boxShadow="0px 4px 16px #00000033"
+        zIndex={3}
+        maxHeight="68px"
+      >
+        <Link
+          p={[3.5, 4]}
+          href="https://microbit.org/code/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Flex>
+            <Box
+              width="3.75rem"
+              fill="#F8F8FB"
+              role="img"
+              aria-label={intl.formatMessage({ id: "visit-dot-org" })}
+            >
+              {brand.squareLogo}
+            </Box>
+            <Box
+              width="212px"
+              role="img"
+              aria-label={intl.formatMessage({ id: "visit-dot-org" })}
+            >
+              {/* Logo needs editing profressionally */}
+              <MicrobitLogo fill="#F8F8FB" />
```

Comment:
Unexpectedly off-white. Let's discuss.

Reply:
I've left this white for now as I think it might be better to do the sidebar off-whites together as we also spotted that the toolkit tabs are off-white now too.